<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>979</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>1111</width>
      <height>421</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>省份</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>日期</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>患者总数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>治愈人数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>现有确诊</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>疑似病例</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>死亡人数</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>省份</string>
    </property>
   </widget>
   <widget class="QComboBox" name="xiala2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>北京市</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>天津市</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>河北省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>山西省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>内蒙古自治区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>辽宁省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>吉林省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>黑龙江省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上海市</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>江苏省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>浙江省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>安徽省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>福建省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>江西省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>山东省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>河南省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>湖北省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>湖南省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>广东省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>广西壮族自治区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>海南省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>重庆市</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四川省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>贵州省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>云南省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西藏自治区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>陕西省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>甘肃省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>青海省</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>宁夏回族自治区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新疆维吾尔自治区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>香港特别行政区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>澳门特别行政区</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>台湾省</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日期</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2020</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>240</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2020</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>140</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1317</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
